ob_v3p0/integrity.md

```diff
@@ -83,7 +79,7 @@ The JWT Payload is the JSON object of the [OpenBadgeCredential](#achievementcred
 
 | Property / Claim Name | Type | Description | Required? |
 | ---------------------- | ---- | ----------- | --------- |
-| \`exp\` | [NumericDate](#numericdate) | The \`validUntil\` property of the OpenBadgeCredential. Required if the OpenBadgeCredential has an \`validUntil\`. | Optional |
+| \`exp\` | [NumericDate](#numericdate) | The expiration time of the  signature. | Optional |
 | \`iss\` | [URI](#uri) | The \`issuer.id\` property of the OpenBadgeCredential. | Required |
```

Comment:
One of the important motiviations of the VC-JOSE-COSE spec was to stop having two ways of encoding the same information - meaning the canonical examples no longer include iss, sub, iat, nbf, exp. 

Therefore I suggest removing all of this table to keep it simpler on organizations. In fact i would consider going the other way and require that implementor not use the conflicting IANA claims which makes verification a lot simpler.